rllib/algorithms/algorithm_config.py

```diff
@@ -296,6 +295,9 @@ def __init__(self, algo_class=None):
         self.auto_wrap_old_gym_envs = True
 
         # `self.rollouts()`
+        # TODO (sven): Clean up the configuration of fully customizable
```

Comment:
We can now publicly configure the class used for rollouts. This used to be configurable before via `config.debugging(worker_cls=..)`, but was not working correctly.